recipe/meta.yaml

```diff
@@ -1,18 +1,18 @@
-{% set version = "6.6.2" %}
+{% set version = "6.6.3" %}
 
 package:
   name: jupyter_console
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/j/jupyter_console/jupyter_console-{{ version }}.tar.gz
-  sha256: 7385dfed8a01fd51e3c98449dae24f38c1ba015d2073835e785671653a0967fc
+  sha256: 566a4bf31c87adbfadf22cdf846e3069b59a71ed5da71d6ba4d8aaad14a53539
 
 build:
   number: 0
-  # jupyter_core does not have support for python 3.7
-  skip: true  # [py<38]
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+  # Upstream supports python 3.7 but would need to use jupyter_core dependency v 4.12 exactly, which we don't have in the channel
+  skip: true  # [py<37]
```

Comment:
if we do not have a support in jupter_core than it would mean we cannot build it using anaconda default channel, so shall we not leave [py<38] as it was before ? I know we were talking about to follow upstream but we do not follow it in dependent package already, right ?

Reply:
not sure either, can we even support builds from non-anaconda channels ? I do not think so, but lets ask someone more experienced

Reply:
I think users can just download and install any package using pip. And then specify the file/channel that contains their package when building jupyter_console using conda-build. So they can mix packages from different channels.

Reply:
yea, on the other hand if we follow that logic we can argue that pinning for C/C++ packages is not required at all as users can build missing dependencies from non-anaconda channels by themselves and anaconda can deliver just recipes  ...

Reply:
No one can build our recipe if we still have `py<37`. Our constraint is `py<38` because of missing `jupyter_core` for python 3.7. That's why `py<38 `is a correct skipping

Reply:
Okay, reverted to skip for py<38.